react-app/ccf/src/App.tsx

```diff
@@ -1,10 +1,71 @@
-import React from 'react';
-import logo from './logo.svg';
+import  { useState, ChangeEvent, useEffect } from 'react';
 import './App.css';
+import { handleUpload, listAndDownloadPDFs } from './functions/storage';
 
 function App() {
+  const [file, setFile] = useState<File | null>(null);
+  const [uploadStatus, setUploadStatus] = useState<string>('');
+  const [downloadStatus, setDownloadStatus] = useState<Array<{ name: string, url: string }>>([]);
+
+  useEffect(() => {
```

Comment:
Loading all pdf's everytime a user loads any page is going to be very resource intensive. We should delete this useEffect before we merge to main.